spec/models/exhibit_spec.rb

```diff
@@ -48,6 +48,71 @@
         end
       end
     end
+
+    describe '#check_publish_change' do
+      before do
+        allow(ExhibitStatusMailer).to receive_message_chain(:exhibit_published, :deliver_later)
+        # disable set_published callback so @was_published can be manually set
+        Spotlight::Exhibit.skip_callback(:save, :before, :set_published)
```

Comment:
You can remove this skip_callback/set_callback setup if you update the ExhibitDefaults concern to rely on the `saved_change_to_published?` check. Instead of setting the instance variable in these tests, you can just create the unpublished or published exhibit depending on the test and update the published attribute.